File: SoloLearn Codes/CMYK to RGB/CmykToRgb.cs
using System;
using System.Linq;

namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            float[] CMYK = new float[4].Select(x => Convert.ToSingle(Console.ReadLine())).ToArray();
            Console.Write(string.Join(",", Enumerable.Range(0, 3).Select(x => 255 * (1 - CMYK[x]) * (1 - CMYK[3])).Select(x => string.Format($"{x:0}"))));
        }
    }
}

File: SoloLearn Codes/CMYK to RGB/cmyk_to_rgb.swift
var CMYK = (0...3).map { x in Float(readLine()!)! }
var RGB = (0...2).map { x in 255 * (1 - CMYK[x]) * (1 - CMYK[3]) }
print(RGB.map { String(Int($0.rounded())) }.joined(separator:","))